src/preview/send.ts

```diff
@@ -0,0 +1,26 @@
+import { send as sendMail } from '../helpers/mail';
+import templates from '../templates';
+import constants from '../helpers/constants.json';
+import { rpcSuccess, rpcError } from '../helpers/utils';
+
+export default async function send(req, res) {
+  const params = {
+    to: constants.example.to,
+    address: [constants.example.address]
```

Comment:
Shouldn't this be `addresses`?

Reply:
Renamed to `addresses`.

As some template uses `address` and other `addresses`, they are now injected conditionally, unless we always inject both (and get rid of the condition)